### PR TITLE
bubblemail: init at 1.2

### PR DESCRIPTION
**Pull request by doronbehar**
###### Motivation for this change

A fork of mailnag, that just works and looks better.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/mailreaders/bubblemail/default.nix [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  strictDeps = false;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  meta = with lib; {
+    description = "A DBus service providing a list of the new and unread user’s mail from:";
+    homepage = "http://bubblemail.free.fr/";
+    license = licenses.gpl2;
```
Comment (jtojnar) [inline pkgs/applications/networking/mailreaders/bubblemail/default.nix:72]: `gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

**Comment by doronbehar** [comment]
@GrahamcOfBorg eval

File: pkgs/applications/networking/mailreaders/bubblemail/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/applications/networking/mailreaders/bubblemail/default.nix]: :+1: :+1: :+1:.

**Comment by doronbehar** [comment]
Can't figure out where that trailing space is coming from.

**Comment by jtojnar** [comment]
pkgs/top-level/all-packages.nix:22850

**Comment by doronbehar** [comment]
Fixed the trailing space directly in master: eaf667bbf00

File: pkgs/applications/networking/mailreaders/bubblemail/default.nix [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  # https://github.com/NixOS/nixpkgs/issues/56943
+  strictDeps = false;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  meta = with lib; {
+    description = "A DBus service providing a list of the new and unread user’s mail from:";
```
Comment (jtojnar) [inline pkgs/applications/networking/mailreaders/bubblemail/default.nix:70]: This looks somewhat weird, otherwise looks good.

Comment (jtojnar) [reply pkgs/applications/networking/mailreaders/bubblemail/default.nix:70]: Maybe `An extensible mail notification service` as used in the repo would work.

Comment (doronbehar) [reply pkgs/applications/networking/mailreaders/bubblemail/default.nix]: Done.

**Review by jtojnar — APPROVED** [review]

**Pull request merged**